scripts/09-history-migration-to-mongo.py

```diff
@@ -0,0 +1,455 @@
+#!/usr/bin/env python
+import json
+import logging
+import socket
+import sys
+import time
+
+import elliptics
+import msgpack
+import pymongo
+
+
+logger = logging.getLogger('mm.convert')
+
+CONFIG_PATH = '/etc/elliptics/mastermind.conf'
+
+try:
+
+    with open(CONFIG_PATH, 'r') as config_file:
+        config = json.load(config_file)
+
+except Exception as e:
+    raise ValueError('Failed to load config file %s: %s' % (CONFIG_PATH, e))
+
+
+def make_meta_session():
+    log = elliptics.Logger('/tmp/ell-namespace-convert.log', config["dnet_log_mask"])
+    node_config = elliptics.Config()
+    meta_node = elliptics.Node(log, node_config)
+
+    addresses = [elliptics.Address(host=str(node[0]), port=node[1], family=node[2])
+                 for node in config["metadata"]["nodes"]]
+    logger.info('Connecting to meta nodes: {0}'.format(config["metadata"]["nodes"]))
+    meta_wait_timeout = config['metadata'].get('wait_timeout', 5)
+
+    try:
+        meta_node.add_remotes(addresses)
+    except Exception as e:
+        logger.error('Failed to connect to any elliptics meta storage node: {0}'.format(
+            e))
+        raise ValueError('Failed to connect to any elliptics storage META node')
```

Comment:
Define error message and pass it to logger and exception.

Do you really need to log here?